x-pack/plugins/observability_solution/metrics_data_access/common/constants.ts

```diff
@@ -5,31 +5,9 @@
  * 2.0.
  */
 
-export const METRICS_INDEX_PATTERN = 'metrics-*,metricbeat-*';
-export const LOGS_INDEX_PATTERN = 'logs-*,filebeat-*,kibana_sample_data_logs*';
-export const METRICS_APP = 'metrics';
-export const LOGS_APP = 'logs';
-
-export const METRICS_FEATURE_ID = 'infrastructure';
-export const LOGS_FEATURE_ID = 'logs';
-
-export type InfraFeatureId = typeof METRICS_FEATURE_ID | typeof LOGS_FEATURE_ID;
-
 export const TIMESTAMP_FIELD = '@timestamp';
-export const MESSAGE_FIELD = 'message';
-export const TIEBREAKER_FIELD = '_doc';
 export const HOST_FIELD = 'host.name';
 export const CONTAINER_FIELD = 'container.id';
 export const POD_FIELD = 'kubernetes.pod.uid';
 
-export const DISCOVER_APP_TARGET = 'discover';
-export const LOGS_APP_TARGET = 'logs-ui';
-
-export const O11Y_AAD_FIELDS = [
-  'cloud.*',
-  'host.*',
-  'orchestrator.*',
-  'container.*',
-  'labels.*',
-  'tags',
-];
+export const METRICS_EXPLORER_API_MAX_METRICS = 20;
```

Comment:
out of curiosity why 20?

Reply:
just saw your comment: https://github.com/elastic/kibana/pull/188112#discussion_r1674317998

Reply:
It's an arbitrary value to limit the number of items. It was aligned with @roshan-elastic  here https://github.com/elastic/kibana/pull/188181#issuecomment-2225235911